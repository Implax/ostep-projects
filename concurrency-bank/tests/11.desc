Correctness test: Single valid transaction (wife)
