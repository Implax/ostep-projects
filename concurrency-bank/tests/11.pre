ln -sf $(pwd)/tests/assets/no_transactions.txt /tmp/Husband.txt && ln -sf $(pwd)/tests/assets/single_transaction.txt /tmp/Wife.txt
