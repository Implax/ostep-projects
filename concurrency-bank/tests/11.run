./bank 690 tests/assets/no_transactions.txt tests/assets/single_transaction.txt