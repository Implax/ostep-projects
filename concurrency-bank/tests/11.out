Opening balance: 690
Withdraw: 600, User: Wife, Account balance after: 90
Closing balance: 90
